config/prow/images-publish-rbac.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
The ProwJobs that publish the infra images, will need to access the `Secret` that contain the key needed for the signin.